mathics/core/systemsymbols.py

```diff
@@ -0,0 +1,27 @@
+# cython: language_level=3
+# -*- coding: utf-8 -*-
+
+from mathics.core.symbols import (
```

Comment:
flake8 informs that none of these imports other than Symbol are used.

Reply:
OK, but I want to have them available for all the modules that import `systemsymbols`... Maybe I should add a comment here...